pages/your_tasks/data_security.md

```diff
@@ -1,26 +1,35 @@
 ---
-title: Data protection
+title: Data security
 contributors: [Pinar Alper, Yvonne Kallberg, Vilem Ded, Eva Csosz, Niclas Jareborg]
-description: How to protect your research data, and how to make research data compliant to GDPR.
-page_id: data_protection
+description: How do you ensure that your data is handled securely.
+page_id: data_security
 related_pages: 
   tool_assembly: [tsd, transmed]
 training:
   - name: Training in TeSS
     registry: TeSS
     url: https://tess.elixir-europe.org/search?q=data+protection#materials
 dsw:
-- name: Will you collect any data connected to a person, "personal data"?
-  uuid: 49c009cb-a38c-4836-9780-8a8b3dd1cbac
-- name: Do you need a Data Protection Impact Assessment?
-  uuid: 8915bd25-db22-4ed6-bcc8-b1bbdc52989e
+- name: Is the risk of information loss, leaks and vandalism acceptably low?
```

Comment:
Where are these DSW links coming from? Because the knowledge model needs to be updated for that

Reply:
Its the Life Sciences DSW Knowledge Model 2.4.4

Do you know where I can submit a MR for this change? https://rdmkit.elixir-europe.org/page_metadata does not say